browser/src/Services/QuickOpen.ts

```diff
@@ -7,18 +7,16 @@
 import { execSync } from "child_process"
 import * as path from "path"
 
+import * as glob from "glob"
```

Comment:
Thanks for finding a replacement for `recursive-readdir`. It's a bummer they never wanted to take that PR / fix the issue around handling errors...